geqie/cli.py

```diff
@@ -124,8 +149,13 @@ def wrapper(*args, **kwargs):
 
 
 def retrieve_options(func) -> Callable:
-    @cloup.option("--encoding", required=True, type=str, help="Name of the encoding from 'encodings' directory")   
-    @cloup.option("--result", required=True, type=str, help="Result from simulation")   
+    @cloup.option(
+        "--encoding",
+        required=True,
+        type=str,
+        help="Encoding folder name under 'encodings/'. (Approval is enforced in Django/worker, not CLI.)",
```

Comment:
Let's not leak the existence of GUI to geqie package.
```suggestion
        help="Name of the encoding from 'encodings' directory.",
```

Reply:
Besides that one small thing its ok